app/lib/primer/join_style_arguments_helper.rb

```diff
@@ -8,7 +8,7 @@ module JoinStyleArgumentsHelper
     # join_style_arguments("width: 100%", "height: 100%") =>
     #   "width: 100%;height: 100%"
     def join_style_arguments(*args)
-      args.compact.join(";")
+      args.compact.join(";").gsub(";;", ";")
```

Comment:
I saw that this `join_style_arguments` can result in extra semicolons being added so this strips it.

Reply:
Hmm why not remove trailing semicolons before joining? Maybe something like

```suggestion
      args.compact.map { |a| a.strip.chomp(";") }.join(";")
```